monkey/monkey_island/cc/ui/src/components/report-components/zerotrust/VennDiagram/index.js

```diff
@@ -0,0 +1,369 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import Dimensions from 'react-dimensions'
+import Tooltip from './Tooltip'
+import CircularNode from './CircularNode'
+import ArcNode from './ArcNode'
+import { TypographicUtilities } from './utility.js'
+import './index.css'
+
+/*
+
+TODO LIST
```

Comment:
Please move this TODO list to the issue #413 and remove it from the code

Reply:
Done

Reply:
Done